Cargo.toml

```diff
@@ -11,7 +11,8 @@ crate-type = ["dylib"]
 
 [dependencies]
 chrono = "0.4"
-tantivy = { git = "https://github.com/tantivy-search/tantivy" }
+#tantivy = { git = "https://github.com/tantivy-search/tantivy" }
+tantivy = {path="/home/paul/git/tantivy"}
```

Comment:
Could we get it back to global? I can't compile and test it on my local requirements dependency